makeflow/src/makeflow.c

```diff
@@ -1599,6 +1626,19 @@ int main(int argc, char *argv[])
 		makeflow_gc_method = MAKEFLOW_GC_ALL;
 	}
 
+	if(wrapper_umbrella && wrapper_umbrella->spec) {
+		debug(D_MAKEFLOW_RUN, "setting dag_node->umbrella_spec...\n");
+		struct dag_node *cur;
+		cur = d->nodes;
+		while(cur) {
+			dag_node_set_umbrella_spec(cur, wrapper_umbrella->spec);
```

Comment:
This seems redundant. I understand having an umbrella spec stored in the node for the eventuality that each node could have a different spec, but when they all share the same umbrella spec it seems to make more sense to only specify it in the umbrella wrapper. Then in the future when node can have different umbrella specs, the dag_node spec is only set when it is different from the standard.